Resources/views/aside/folder.html.twig

```diff
@@ -0,0 +1,12 @@
+{% if treeChildItems|length > 0 %}
+    <ul>
+        {% for treeChildItem in treeChildItems %}
+             <li><a href="{{ path( 'ez_urlalias', { 'locationId': treeChildItem.contentInfo.mainLocationId } ) }}" title="{{ ez_content_name( treeChildItem.contentInfo ) }}">{{ ez_content_name( treeChildItem.contentInfo ) }}</a></li>
+        {% endfor %}
+    </ul>
+{% elseif isChild is not defined %}
+    <li>
+        <a href="{{ path( 'ez_urlalias', { 'locationId': location.contentInfo.mainLocationId } ) }}" title="{{ ez_content_name( location.contentInfo ) }}">{{ ez_content_name( location.contentInfo ) }}</a>
+        {{ render_esi( controller( 'eZDemoBundle:Folder:showFolderListAsideView', { 'locationId': location.contentInfo.mainLocationId } ) ) }}
```

Comment:
IMHO it is better practice to use `ez_content:viewLocation` and configure controller with the view.
The template could then be general as it renders only linked name, so nothing folder specific.